crates/api_models/src/payment_methods.rs

```diff
@@ -144,6 +144,16 @@ pub struct PaymentMethodResponse {
     pub created: Option<time::PrimitiveDateTime>,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
+pub struct CardDetailsPaymentMethod {
```

Comment:
How are we planning to store bank details here?

Reply:
Not the purpose of this PR. Probably can be taken up in another PR.